src/server/contentProvider.ts

```diff
@@ -34,49 +34,52 @@ export class ContentProvider implements TextDocumentContentProvider {
 
         // tslint:disable-next-line:no-http-string
         const uri = `http://localhost:${port}/?_&${queryArgs.join('&')}`;
-        this.serviceContainer.getAll<ILogService>(ILogService).forEach(logger => {
-            logger.log(`Server running on ${uri}`);
-        });
+        this.serviceContainer.getAll<ILogService>(ILogService)
+            .forEach(logger => {
+                logger.log(`Server running on ${uri}`);
+            });
 
         return `
                     <!DOCTYPE html>
-                    <head><style type="text/css"> html, body{ height:100%; width:100%; overflow:hidden; padding:0;margin:0; } </style>
-                    <titleCan I give a title</title>
-                    <script type="text/javascript">
-                        function start(){
+                    <head>
+                        <meta http-equiv="Content-Security-Policy" content="default-src * 'unsafe-inline' 'unsafe-eval'; script-src * 'unsafe-inline' 'unsafe-eval'; connect-src * 'unsafe-inline'; img-src * data: blob: 'unsafe-inline'; frame-src *; style-src * 'unsafe-inline';" />
```

Comment:
Please add a property content security policy. At this point, you may as well not even have one

Reply:
@mjbvz Sorry? the meta tag is already defined. Anything else I need to define? HTTP headers?

Reply:
The current policy allows everything. This suppresses the warning in VS Code but does not bring any actually security benefits. 

See [docs like this](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) for help writing a proper csp.

Reply:
I do know that this does not bring much security, but first I wanted to know why it does not display anything using macOS only...

Reply:
For mac the webView port may always be different from the extensionHost port